### DISCUSSION TITLE
Where do I get the SHA256 of a specific commit point from?

### DISCUSSION THREAD

**Discussion by aaomidi** [tweag/rules_nixpkgs#590] [Q&A]
I'm not sure where to get the SHA256 value

**Comment by avdv** [comment]
I assume you're using `http_archive`?!

You could just leave it out. Bazel will print something like this:
```
DEBUG: Rule 'io_tweag_rules_nixpkgs' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "6f763feca1a7526f30177a7a685b9c24beee5ca6df9c47358e885b02e50ffe61"
```
You could also manually download the file given in the `url` attribute to the `http_archive` and compute the hash using `sha256sum`.
